README.md

```diff
@@ -1209,24 +1214,27 @@ But this window is not always open.
 
 ### Kinds of stock options
 
-☝️ 🄳 Compensatory stock options come in two flavors, **incentive stock options (or ISOs)**
-and **non-qualifying stock options (or NQOs, or NSQOs)**. Confusingly, lawyers can’t even
-agree on the names of these two kinds of stock options, so you’ll also see them called
+🄳 Compensatory stock options come in two flavors, **incentive stock options (or ISOs)** and
+**non-qualifying stock options (or NQOs, or NSQOs)**. Confusingly, lawyers can’t even agree
+on the names of these two kinds of stock options, so you’ll also see them called
 **statutory stock options** and **non-statutory stock options (or NSOs)**, respectively.
 
 🚧 Infographic:
 A diagram with all the terms is likely worth it.
 
-ISOs are common for employees because they have the possibility of being more favorable
-from a tax point of view than NSOs.
-They can only be granted to employees (not independent contractors or directors who are
-not also employees).
-But ISOs have a number of limitations and conditions and can also create difficult tax
-consequences. We’ll get into all this [after we cover taxes](#taxes-on-isos-and-nsos).
+- Companies generally decide to give ISOs or NSOs depending on the legal advice they get.
+  It’s rarely up to you which you get, so you need to know about both.
```

Comment:
It is also possible to have both at once! And if you leave the company and are allowed to keep your shares (or you die and your shares revert to your estate) your ISOs will convert to NSOs. ISOs can also convert to NSOs in some states that do not consider you employed when you take a leave of absence. Just more reasons it's important to know about both.

Reply:
Agree, good point. Will add a note.